spatialdata/_core/_spatialdata.py

```diff
@@ -200,6 +244,20 @@ def is_backed(self) -> bool:
 
     # TODO: from a commennt from Giovanni: consolite somewhere in a future PR (luca: also _init_add_element could be cleaned)
     def _get_group_for_element(self, name: str, element_type: str) -> zarr.Group:
+        """Get the group for an elemnt, creates a new one if the element doesn't exist
+
+        Parameters
+        ----------
+        name : str
+            name of the element
+        element_type : str
+            type of the element. Should be in ["images", "labels", "points", "polygons", "shapes"].
+
+        Returns
+        -------
+        zarr.Group
+            either the exisitng sub-group or a new one
```

Comment:
same here, this one should just be
`:func:`zarr.hierarchy.group` following the definition in e.g. [here](https://zarr.readthedocs.io/en/stable/api/hierarchy.html)